lnbits/core/views/callback_api.py

```diff
@@ -33,3 +41,104 @@ async def api_generic_webhook_handler(
         success=False,
         message=f"Unknown fiat provider '{provider_name}'.",
     )
+
+
+async def handle_stripe_event(event: dict):
```

Comment:
i think this code should be somewhere under `services` not `views`

Reply:
maybe `servces/callback.py` ?

Reply:
`services/callback.py` doesn;t say much. Any other suggestion?